### ISSUE TITLE
Explore crossbuild gradle plugin

### ISSUE THREAD

**Issue by Jolanrensen** [Kotlin/kotlin-spark-api#179]
https://github.com/prokod/gradle-crossbuild-scala/pull/117

**Comment by asm0dey** [comment]
@Jolanrensen is it implemented on the side of the crossbuild plugin?

**Comment by Jolanrensen** [comment]
@asm0dey Yes, but I haven't had time to try it in the project here yet, especially since the GH actions approach works so well now

**Comment by asm0dey** [comment]
Don't you think this could improve the overall performance?

**Comment by Jolanrensen** [comment]
hmm maybe, but not by much. Only if you want to quickly test a couple of versions locally it would help. There would be no difference with GH actions, since that runs in parallel anyways (and gh actions can also be run locally). Eventually, the crossbuild plugin will be better, but it will cost a lot of time to figure out how to rework the entire build schema again and I'm not sure if that's worth it right now. What do you think?
